app/controllers/idv/in_person/public/usps_locations_controller.rb

```diff
@@ -3,15 +3,35 @@
 module Idv
   module InPerson
     module Public
+      class UspsLocationsError < StandardError
+        def initialize
+          super('Unsupported characters in address field.')
+        end
+      end
+
       class UspsLocationsController < ApplicationController
         skip_forgery_protection
 
+        include IppHelper
+
+        rescue_from ActionController::InvalidAuthenticityToken,
+                    Faraday::Error,
+                    StandardError,
+                    UspsLocationsError,
+                    Faraday::BadRequestError,
+                    with: :handle_error
+
         def index
           candidate = UspsInPersonProofing::Applicant.new(
             address: search_params['street_address'],
             city: search_params['city'], state: search_params['state'],
             zip_code: search_params['zip_code']
           )
+
+          unless candidate.has_valid_address?
+            raise UspsLocationsError.new
```

Comment:
What's the benefit of using `raise UspsLocationsError.new` instead of `raise UspsLocationsError`?  (I'm asking because I'm curious, I'm not asking because I think it needs to be changed.  I see both patterns being used in our codebase.)

Reply:
Ooh also a great question. These sources offer an explanation that they are essentially the same: https://blog.arkency.com/ruby-raise-exception-dot-new-or-raise-exception-theyre-both-the-same/
https://stackoverflow.com/questions/31373514/difference-between-rails-raise-standarderror-new-and-raise-standarderror

I think I'll use the simpler one!

Reply:
Sounds good.  Thanks for linking the blog. 🙏🏻